app/src/polkadot_types.c

```diff
@@ -481,7 +677,7 @@ parser_error_t _toStringBalanceOf(
         char *outValue,
         uint16_t outValueLen,
         uint8_t pageIdx,
-        uint8_t *pageCount) {
+        uint8_t *pageCount) {   
     MEMSET(outValue, 0, outValueLen);
     MEMSET(bufferUI, 0, sizeof(bufferUI));
```

Comment:
Ok, here the toString is clear. I forgot about this.
Maybe it is a good idea to create a #define BALANCEOF_LEN 16 or similar to link the _read and the 16 in `bignumLittleEndian_to_bcd(bcdOut, bcdOutLen, v->_ptr, 16);`